lib/api/apiUtils/authorization/permissionChecks.js

```diff
@@ -385,10 +387,19 @@ function processBucketPolicy(requestType, bucket, canonicalID, arn, bucketOwner,
  * @returns {boolean} - true if the quota is valid, false otherwise
  */
 async function validateQuotas(bucket, account, apiNames, apiMethod, inflight, log, callback) {
```

Comment:
probably not in this PR, but this function is very weird: being both async and with a callback...

Reply:
As discussed I will callbackify the call to scubaclient in the PR this one is based on

Reply:
Callbackify was done in the base PR.